src/components/m-dropdown/index.scss

```diff
@@ -46,7 +45,6 @@
 // js enhanced dropdown
 .m-dropdown__toggle {
   @include button-reset();
-  @include typo-text-longer();
```

Comment:
Same here, typography is mandatory

Reply:
I removed all datepicker changes. they were not meant to be in this PR.